Algorithms/src/main/java/dynamic_programing/Factorial.java

```diff
@@ -0,0 +1,10 @@
+public class Factorial {
```

Comment:
The build is failing for the duplicate class that it found for factorial, can you please take a look into it?
```
/home/runner/work/Let-Us-Java/Let-Us-Java/Algorithms/src/main/java/numbers/Factorial.java:[1,8] duplicate class: Factorial
```

Reply:
im fixing it right now, i did'nt noticed actually that i had created two classes with same name.